### DISCUSSION TITLE
Codespace creation still failing

### DISCUSSION THREAD

**Discussion by savwatson13** [community/community#26706] [Codespaces]
<p>The issue says resolved on the site but the codespace still isn’t loading for me.</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Hello <a class="mention" href="/savwatson13">@savwatson13</a>!</p>
<p>I see that you recently joined the community and posted a topic about Codespaces Beta. I wanted to introduce you to a recent change that you may not have yet seen: <a href="https://github.community/t/codespaces-beta-expands-to-discussions/165089">Codespaces Beta Expands to Discussions </a></p>
<p>Please take a look for the new public location in order to continue this conversation with our team. Thank you again for being here!</p>
<blockquote>
<p>The Codespaces Beta category has moved over to Discussions! Our Codespaces team is eager to continue in conversations and knowledge-sharing with you as you share your questions and feedback with us. Please head over to the new public space for <a href="https://github.com/github/feedback/discussions/categories/codespaces-feedback">Codespaces Feedback</a> to take a look!</p>
</blockquote>
